pkg/extensions/extension_gqlplayground.go

```diff
@@ -0,0 +1,26 @@
+//go:build search && gqlplayground
+// +build search,gqlplayground
```

Comment:
@rchincha Shall we include the gqlplayground together with the search extension instead?

Actually I'm thinking there should be a single endpoint for both search and config, and the playground should cover the queries and documentation of functionalities for both.